dot/types/babe_digest.go

```diff
@@ -119,5 +119,19 @@ func NewBabeSecondaryVRFPreDigest(authorityIndex uint32,
 	}
 }
 
+// ToPreRuntimeDigest returns the BabeSecondaryVRFPreDigest as a PreRuntimeDigest
+func (d *BabeSecondaryVRFPreDigest) ToPreRuntimeDigest() (*PreRuntimeDigest, error) {
+	digest := NewBabeDigest()
+	err := digest.Set(*d)
+	if err != nil {
+		return nil, err
+	}
+	enc, err := scale.Marshal(digest)
```

Comment:
nit `babeDigestEncoding` would be nice. I'm an explicit/full words maniac so let me know if i'm bothering as well 😅